ui/appui-react/src/appui-react/selection/HideIsolateEmphasizeManager.ts

```diff
@@ -330,8 +330,20 @@ export class HideIsolateEmphasizeManager extends HideIsolateEmphasizeActionHandl
     vp.changeCategoryDisplay(ids, true);
   }
 
+  private static async getSelectionSetElementProps(iModel: IModelConnection) {
```

Comment:
Something like `toPersistentSet(elements: Id64Set): Id64Set` might be easier to understand, reuse and test.

Reply:
Usually I would agree but I think in this case that is not needed because we can't test this method either way (because it is private) and for reusability I like to use YAGNI (you ain't gonna need it), so if there is no need for reusability now or in the near future, why do it. The methods is perfect for those two other places where it is called. This is just my opinion though.